WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,6 +1,4 @@
 * [*] Site previews: Reduced visual flickering when previewing sites and templates. [#17861]
 * [*] Stats: Scroll to new Insights card when added. [#17894]
 * [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
-* [*] [Jetpack-only] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
```

Comment:
This should NOT have been deleted from the Jetpack Release Notes. On the contrary, that's one key entry that should be in the Jetpack notes but not on the WP ones